packages/supi/src/symlinkDependencyTo.ts

```diff
@@ -4,6 +4,13 @@ import symlinkDir = require('symlink-dir')
 
 export default function symlinkDependencyTo (alias: string, peripheralLocation: string, dest: string) {
   const linkPath = path.join(dest, alias)
+  // Don't symlink deps from packages that are outside of the monorepo root.
+  //   These may be symlinks from another project, and we don't want to break the other project.
+  // TODO(vjpr): Is `process.cwd()` a reliable way to get the monorepo root?
+  if (!linkPath.startsWith(process.cwd())) {
+    // TODO(vjpr): Should print some kind of warning maybe?
+    return { reused: true } // TODO(vjpr): Not sure we should misuse reused like this.
```

Comment:
I think returning this is misleading. I think maybe the check should happen not inside this function. Maybe this function shouldn't even be called

Reply:
So `symlinkDependencyTo` is used in three places.

![screen shot 2018-11-01 at 10 48 20 am](https://user-images.githubusercontent.com/281413/47844895-d2cfe000-ddc3-11e8-84b9-5438109dcfad.png)

You would suggest moving the code to these three locations?

Reply:
> reused

`reused` it only used by 1 usage:

```
if (
  !opts.dryRun &&
  (await symlinkDependencyTo(rootAlias, depGraphNode.peripheralLocation, modulesDir)).reused
) return
```

It seems to be only used to prevent a log message printed if link was reused.

Making `symlinkDependencyTo` return `linkCreated` instead of `reused` (forwarded from `symlinkDir`) would be better.

Reply:
But I think for now, `reused` can be used for this purpose...but the name may cause misuse later...

Reply:
Also, if its a `dryRun`, shouldn't we still check for `reused`, and inside `symlinkDepTo` we should conditionally run `symlinkDir` if dryRun is false?